examples/inspection/plot_partial_dependence.py

```diff
@@ -1,74 +1,82 @@
 """
-========================
-Partial Dependence Plots
-========================
+===============================================================
+Partial Dependence and Individual Conditional Expectation Plots
```

Comment:
Unfortunately the plot changes in this example have broken the plot in the User Guide (currently at https://102291-843222-gh.circle-artifacts.com/0/doc/modules/partial_dependence.html)